website/src/views/login.js

```diff
@@ -29,9 +29,9 @@ export default function Login() {
         axios.get(`/api/v2/login`, {
             headers: { 'Authorization': token }
         }).then((loginRes) => {
-            console.log(loginRes);
+            console.log(`Login result: ${loginRes}`);
```

Comment:
```suggestion
```
I need to configure an autolinter to run on commits, but we shouldn't be logging anything that isn't 100% necessary in prod.